src/canvasMachine.typegen.ts

```diff
@@ -0,0 +1,28 @@
+// This file was automatically generated. Edits will be overwritten
+
+export interface Typegen0 {
+  '@@xstate/typegen': true;
+  eventsCausingActions: {
+    assignAfterZoomIn: 'ZOOM.IN';
+    resetPosition: 'POSITION.RESET';
+    setPositionAfterSourceChanged: 'SOURCE_CHANGED';
+    assignZoomAfterFitToContent: 'FIT_TO_CONTENT';
+    assignViewBoxAfterFitToContent: 'FIT_TO_CONTENT';
+    persistPositionToLocalStorage: string;
```

Comment:
this is definitely wrong, cause like we've discussed - this should always be a union of string literals (or a single string literal)

Reply:
Yep, this would be different if I did it with the extension now